ffi/capi/dart/package/pubspec.yaml

```diff
@@ -4,10 +4,10 @@ description: Dart bindings for ICU4X.
 repository: https://github.com/unicode-org/icu4x/tree/main/ffi/capi/dart/package
 
 environment:
-  sdk: '>=2.18.0'
+  sdk: '>=2.18.0 <4.0.0'
 
 dependencies:
-  ffi: ^2.1.0
+  ffi: '>=1.1.0 <3.0.0'
```

Comment:
I don't actually know if this is true

Reply:
You mean if versions `2.x` are fine to use?

Reply:
yes

Reply:
Yes, if the tests pick 2.x then 2.x works, if they pick 1.x then 1.x works, hard to tell

Reply:
I tested it with 2.x . In general, when giving a large range of version constraints, one should test using `dart pub upgrade` and `dart pub downgrade` to make sure both work. It's easier to just depend on the latest version to not have these issues, so `^2.1.0` in this case. Or is there a reason to stay with `1.x`?

Reply:
Ideally we'd restrict downstream clients as little as possible. I don't know how Dart dependency resolution works, but they might be in a situation where they can only use 1.x?

Reply:
Our policy in general is update as soon as major version releases are available, as this shouldn't happen. I adapted the deps to reflect that.